khmer/_oxli/graphs.pxd

```diff
@@ -85,6 +85,43 @@ cdef extern from "oxli/hashtable.hh" namespace "oxli":
     cdef cppclass CpNodetable "oxli::Nodetable" (CpHashtable):
         CpNodetable(WordLength, vector[uint64_t])
 
+    cdef cppclass CpQFCounttable "oxli::QFCounttable":
```

Comment:
this should be derived from `CpHashtable` (see above with Nodetable and Counttable), which will allow you to omit all these redefinitions